src/v0/util/facebookUtils/networkHandler.js

```diff
@@ -195,6 +195,9 @@ const errorDetailsMap = {
       .setMessage('There have been too many calls to this ad-account.')
       .build(),
   },
+  200: {
+    default: new ErrorDetailsExtractorBuilder().setStatus(403).setMessageField('message').build(),
```

Comment:
can we put something else instead of 'message'

Reply:
No, it contains the actual error message.

Reply:
Got it